src/test/java/com/flightstats/hub/dao/aws/S3AccessMonitorTest.java

```diff
@@ -5,23 +5,28 @@
 import com.amazonaws.services.s3.model.PutObjectRequest;
 import com.amazonaws.services.s3.model.PutObjectResult;
 import com.amazonaws.services.s3.model.S3Object;
+import com.flightstats.hub.config.AppProperty;
+import com.flightstats.hub.config.PropertyLoader;
+import com.flightstats.hub.config.S3Property;
 import com.flightstats.hub.dao.Dao;
 import com.flightstats.hub.model.ChannelConfig;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.IOException;
 
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
-import static org.junit.Assert.assertFalse;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 public class S3AccessMonitorTest {
-    private final S3BucketName s3BucketName = new S3BucketName("test", "bucket");
+    private final S3BucketName s3BucketName = new S3BucketName(
```

Comment:
same question about mockability here. Looks like we're mocking the hub s3 client, so I have hope that it's a reasonable thing here.